### PR TITLE
Lock theme classic version to 2.0.8

### PR DESCRIPTION
**Pull request by matthieu-rolland**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.0.x
| Description?      | Lock theme classic version to 2.0.8
| Type?             | improvement
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | none
| Fixed ticket?     | 
| Related PRs       | 
| Sponsor company   | PrestaShop SA

### CODE REVIEW

**Comment by matthieu-rolland** [comment]
I wonder if I should set `prestashop/classic` to `>=2.0,<2.1` in composer.json as well ?

File: composer.json [hunk 1]
```diff
@@ -57,7 +57,7 @@
         "prestashop/blockreassurance": "^5.1",
         "prestashop/blockwishlist": "^2.0",
         "prestashop/circuit-breaker": "^4.0",
-        "prestashop/classic": "^2.0",
+        "prestashop/classic": ">=2.0,<2.1",
```
Comment (kpodemski) [inline composer.json]:
```suggestion
        "prestashop/classic": "~2.0.0",
```
i think it's the same? or `2.0.*`? :D

Comment (matthieu-rolland) [inline composer.json]: I had to change this now that 2.1.0 exists, otherwise it would update to 2.1.0

Comment (matthieu-rolland) [reply composer.json]: it was the only syntax I was sure about by memory :D 
I'll check yours and update :+1:

**Review by kpodemski — DISMISSED** [review]

**Review by matks — DISMISSED** [review]

**Review by jolelievre — DISMISSED** [review]

**Comment by kpodemski** [comment]
@matthieu-rolland something went wrong with composer/json/lock, cs-fixer complains 🤔

**Review by matks — DISMISSED** [review]

**Review by kpodemski — DISMISSED** [review]

**Review by kpodemski — DISMISSED** [review]

**Comment by kpodemski** [comment]
@matthieu-rolland maybe something went wrong with some other PRs and we should fix this issue reported by cs-fixer? 🤔

**Comment by matthieu-rolland** [comment]
> @matthieu-rolland maybe something went wrong with some other PRs and we should fix this issue reported by cs-fixer? thinking

yes just did that, strangely cs-fixer raised no issue locally...

**Review by kpodemski — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]
Thanks @matthieu-rolland

**Comment by AureRita** [comment]
Hi @matks 

Thank for your PR, I tested it and it seems to works correctly. 

Thank you !

**Review by AureRita — APPROVED** [review]

**Comment by kpodemski** [comment]
🎉

**Pull request merged**